Readme.md

```diff
@@ -44,6 +44,7 @@ Format the given value in bytes into a string. If the value is negative, it is k
 | thousandsSeparator | `string`｜`null` | Example of values: `' '`, `','` and `.`... Default value to `''`. |
 | unit | `string`｜`null` | The unit in which the result will be returned (B/KB/MB/GB/TB). Default value to `''` (which means auto detect). |
 | unitSeparator | `string`｜`null` | Separator to use between number and unit. Default value to `''`. |
+| outputAsObject | `boolean`｜`null` | Whether to return a string, or an object that returns a unitless value and the unit. Default value to `false` |
```

Comment:
I see the idea you're trying to achieve but I don't like the interface you used for it -- varying the output type based on an option can lead to surprising things. Why not simply make this as a new method altogether? 'm not sure. The title of this section and the corresponding jsdoc no longer have the correct return type.

In reality, it seems like you could simply split the output on the `unitSeparator` anyway, like `bytes.format(24211, { unitSeparator: '|' }).split('|')`